src/components/ShareTrajectoryModal/style.css

```diff
@@ -25,28 +45,62 @@
     width: 395px;
 }
 
-.copy-button {
+.embed-header {
     display: flex;
+    justify-content: space-between;
+}
+
+.settings-button {
+    border: none;
+    background-color: var(--light-theme-bg);
+    font-size: 16px;
+    cursor: pointer;
+}
+
+.embed-settings {
+    transition: height 0.3s;
+    height: 0px;
+}
+
+.embed-settings.open {
+    margin-bottom: 10px;
+    height: 56px;
+}
+
+.accent-text {
+    font-size: 16px;
+    font-weight: 600;
+}
+
+.proportion-settings {
     align-items: center;
-    gap: 6px;
+    display: flex;
+    gap: 8px;
 }
 
-.copy-button span:global(.anticon) {
-    margin: 0px;
-    padding-top: 2px;
+.number-inputs {
+    text-align: right !important;
```

Comment:
can this just be targeted more directly instead of using an important?

Reply:
was able to remove!